src/main/java/mate/academy/store/model/RoleName.java

```diff
@@ -0,0 +1,6 @@
+package mate.academy.store.model;
+
+public enum RoleName {
+    ADMIN,
```

Comment:
About your question, if it's still actual. 
Spring Security automatically added prefix ROLE_ if you use hasRole() method. 
Try to debug your code, maybe you need to create ROLE_ADMIN, ROLE_USER here that Spring could find in DB particular role with prefix 
it's just an opinion, need to debug your code

Reply:
> About your question, if it's still actual. Spring Security automatically added prefix ROLE_ if you use hasRole() method. Try to debug your code, maybe you need to create ROLE_ADMIN, ROLE_USER here that Spring could find in DB particular role with prefix it's just an opinion, need to debug your code

As far as I understand, I have to replace USER and ADMIN with ROLE_ADMIN and ROLE_USER everywhere. I have already tried this but it was unsuccessful. In addition, I found that I cannot register a new user in postman, again I get a 401